src/smc-util/message.coffee

```diff
@@ -798,12 +850,12 @@ logged in.
 # return a JSON object with all data that is
 # meant to be publically available about this project,
 # who owns it, the title/description, etc.
-API message
+API message # unimplemented
```

Comment:
why  unimplemented ?

If anything is genuinely unimplemented, then it should just be deleted.